lib/pbench/server/api/resources/query_apis/iteration_samples.py

```diff
@@ -111,10 +110,10 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
             "path": f"/{uri_fragment}/_search?scroll=1m",
             "kwargs": {
                 "json": {
-                    "size": 100,
+                    "size": 100,  # The maximum default size for scrolling is 10000
```

Comment:
I don't think that comment has the right phrasing.  The [ES documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/scroll-api.html) puts it thusly:

> We no longer recommend using the scroll API for deep pagination. If you need to preserve the index state while paging through more than 10,000 hits, use the `search_after` parameter with a point in time (PIT).

so, 10,000 isn't the "maximum default", it's more like the upper limit on what you can reach via scrolling....